src/skimage/_shared/utils.py

```diff
@@ -1096,3 +1097,121 @@ def as_binary_ndarray(array, *, variable_name):
                 f"safely cast to boolean array."
             )
     return np.asarray(array, dtype=bool)
+
+
+def _prescale_value_range(image, *, mode, stacklevel=3):
+    """Scale the value range of `image` according to the selected `mode`.
+
+    Parameters
+    ----------
+    image : ndarray
+        The image to scale.
+    mode : {'minmax', 'legacy', False} or tuple[float, float], optional
+        Controls the rescaling behavior for `image`.
+
+        ``'minmax'``
+            Normalize `image` between 0 and 1 regardless of dtype. After
+            normalization its minimum and maximum values will be 0 and 1
+            respectively. This is a shorthand for
+            ``prescale=(image.min(), image.max())``.
+
+        ``'legacy'``
+            Normalize only if `image` has an integer dtype, if `image` is of
+            floating dtype, it is left alone. See :ref:`.img_as_float` for
+            more details.
+
+        ``(lower, higher)``
+            Normalize `image` such that ``lower`` and ``higher`` are scaled
+            with ``(img - lower) / (higher - lower)``
+            to 0 and 1 respectively.
+
+        ``False``
```

Comment:
I have a vague feeling we agreed this should be a string like `'none'`, but I can't remember the reasoning.

Reply:
@jni - I have some vague memory that you had a preference for the string `'none'` here.   Is that right?

Reply:
> I'm fine with `none` to constrict the types.

Do you mean `None`?